README.md

```diff
@@ -11,70 +11,28 @@ We are designing a graph-manipulating DSL that exposes functionality that pangen
 Our DSL will support graph queries in the vein of the [odgi][] project.
 We will compile programs written in this DSL into the [Calyx][] IR and then leverage Calyx to generate hardware accelerators.
 
+Installation using Docker
+-------------------------
+Installation is easy if you use our Docker [package][]:
```

Comment:
I know this is a ridiculous nitpick, but I'd shy away from calling the Docker path "installation"… we're not really installing anything anywhere; just running a container where everything is pre-installed. So maybe we should just call this section "Running using Docker" or similar?